algorithms/max-flow/problems/cowex/Makefile

```diff
@@ -0,0 +1,9 @@
+CXX=clang++
```

Comment:
Should we really be using `clang++` here? It should be fine, but the rest of the problems use contest standards.